nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java

```diff
@@ -383,7 +383,7 @@ public static String getOntology(URI baseUri) {
 
     @JsonIgnore
     public boolean isPublishable() {
-        return !DRAFT_FOR_DELETION.equals(getStatus()) && hasMainTitle() && hasReferencedContent();
+        return !DRAFT_FOR_DELETION.equals(getStatus()) && hasMainTitle();
```

Comment:
Om jeg husker rett så har jeg fjernet denne sjekken et sted i publication-api på PublishingRequestCase opprettet.
Terje/Jan Eirik burde finne ut av hva som gjør publikasjonen publishable. For denne gjør at man kan publisere publikasjon med masse metadata, og så fjerne alt bortsatt fra tittel og publisere, som er litt dumt.

Reply:
Altså denne funksjonaliteten hindrer at man endrer på filene på en ting som er publisert (som i indexert)? Altså flyt: "registrere -> publisere -> fjerne fil" blir nå mulig?

Reply:
Per nå så kan man opprette completed publishingRequest ticket uten noen associated artefacts, men posten blir aldri publisert i søke-indexen.

Reply:
Her strider vi i mot tidligere lære, men jeg tenker at du vet noe jeg ikke vet